cmake/defaults/Packages.cmake

```diff
@@ -227,6 +230,9 @@ if (PXR_BUILD_IMAGING)
             cmake_policy(SET CMP0072 OLD)
         endif()
         find_package(OpenGL REQUIRED)
```

Comment:
shouldn't the find_package be removed to avoid getting the full path and also the -l bit?

Reply:
I'd left it for Linux/Windows and then special cased the specific variable used for macOS. 

I could move the find_package behind a platform check so it never runs for macOS, but it shouldn't be meaningfully different I think?